add newsletter no 23 as blog post (#1261)
questdb/questdb.io: blog/2023-01-05-newsletter-23.md

---

> title: 2022 in review and some exciting news

Reviewer: ```suggestion
title: 2022 in Review and Exciting News about 2023
```

author: QuestDB
author_url: https://github.com/questdb
author_image_url: https://avatars.githubusercontent.com/u/52297642?s=200&v=4
description:
  "Happy new year! Let's celebrate some of our achievements and begin the new
  year with some exciting news: QuestDB 7.0 & Cloud GA coming soon, Javier
  hosting the Fast and streaming data devroom at FOSDEM and more!"
tags: [newsletter]
image: /img/blog/newsletter-preview.png

> slug: newsletter-2022-in-review

Reviewer: ```suggestion
slug: newsletter-questdb-2022-in-review
```

> slug: 2022-in-review-exciting-news

Reviewer: ```suggestion
slug: newsletter-2022-in-review
```

> slug: newsletter-questdb-2022-in-review

Reviewer: ```suggestion
slug: newsletter--2022-achievements-codetothemoon-feature
```

> slug: newsletter-2022-rankings-codetothemoon-youtube

Reviewer: ```suggestion
slug: newsletter-uuid-pandas-dataframes-cloud-codetothemoon-youtube
```

> slug: newsletter-2022-achievements-codetothemoon-feature

Reviewer: ```suggestion
slug: newsletter-2022-rankings-codetothemoon-youtube
```

> slug: newsletter--2022-achievements-codetothemoon-feature

Reviewer: ```suggestion
slug: newsletter-2022-achievements-codetothemoon-feature
```

---

import Banner from "@theme/Banner"

<Banner
  alt="Image of QuestDB logo"
  src="/img/blog/newsletter.png"
  width={692}
  height={200}
/>

Happy new year! Let's celebrate some of our achievements and begin the new year
with some exciting news: QuestDB 7.0 & Cloud GA coming soon, Javier hosting the
Fast and streaming data devroom at FOSDEM and more!

> 

Reviewer: ```suggestion
<!--truncate-->
```
